chroma_api/host.py

```diff
@@ -138,7 +138,7 @@ class ClientMountResource(ChromaModelResource):
     # This resource is only used for integration testing.
 
     host = fields.ToOneField("chroma_api.host.HostResource", "host")
-    filesystem = fields.ToOneField("chroma_api.filesystem.FilesystemResource", "filesystem")
+    filesystem = fields.CharField()
```

Comment:
This is why the tests were failing. Note that this is unrelated to running in production ("This resource is only used for integration testing"). What do you think about rewriting the tests without this resource? We could just get filesystem details and number of mounts from there, I think.

Reply:
This resource is also available in production. If anything that comment should be removed as it's not accurate.